test/math/FloatSize2DTest.ooc

```diff
@@ -90,10 +90,62 @@ FloatSize2DTest: class extends Fixture {
 		})
 		this add("int casts", func {
-			vector := vector0 toIntSize2D()
+			vector := this vector0 toIntSize2D()
 			expect(vector width, is equal to(22))
 			expect(vector height, is equal to(-3))
 		})
+		this add("minimum maximum", func {
+			_max := this vector0 maximum(this vector1)
+			_min := this vector0 minimum(this vector1)
+			expect(_max width, is equal to(22.221f) within(this precision))
+			expect(_max height, is equal to(13.1f) within(this precision))
+			expect(_min width, is equal to(12.221f) within(this precision))
+			expect(_min height, is equal to(-3.1f) within(this precision))
+		})
+		this add("rounding", func {
+			_round := this vector1 round()
+			_ceiling := this vector1 ceiling()
+			_floor := this vector1 floor()
+			expect(_round width, is equal to(12.0f) within(this precision))
+			expect(_round height, is equal to(13.0f) within(this precision))
+			expect(_ceiling width, is equal to(13.0f) within(this precision))
+			expect(_ceiling height, is equal to(14.0f) within(this precision))
+			expect(_floor width, is equal to(12.0f) within(this precision))
+			expect(_floor height, is equal to(13.0f) within(this precision))
+		})
+		this add("p norm", func {
+			oneNorm := this vector0 pNorm(1.0f)
+			euclideanNorm := this vector0 pNorm(2.0f)
+			expect(oneNorm, is equal to(25.321f) within(this precision))
+			expect(euclideanNorm, is equal to(22.436f) within(0.01f))
+		})
+		this add("clamp", func {
+			clamped := this vector1 clamp(this vector0, this vector2)
+			expect(clamped width, is equal to(22.221f) within(this precision))
+			expect(clamped height, is equal to(10.0f) within(this precision))
+		})
+		this add("scalar product", func {
+			expect(this vector0 scalarProduct(this vector1), is equal to (230.95f) within(0.01f))
+		})
+		this add("interpolation", func {
+			interpolate1 := FloatSize2D linearInterpolation(this vector0, this vector1, 0.0f)
+			interpolate2 := FloatSize2D linearInterpolation(this vector0, this vector1, 0.5f)
+			interpolate3 := FloatSize2D linearInterpolation(this vector0, this vector1, 1.0f)
+			expect(interpolate1 width, is equal to(this vector0 width) within(this precision))
+			expect(interpolate1 height, is equal to(this vector0 height) within(this precision))
+			expect(interpolate2 width, is equal to(17.22f) within(0.01f))
+			expect(interpolate2 height, is equal to(5.0f) within(0.01f))
+			expect(interpolate3 width, is equal to(this vector1 width) within(this precision))
+			expect(interpolate3 height, is equal to(this vector1 height) within(this precision))
+		})
+		this add("area, length, empty", func {
+			_empty := FloatSize2D new()
+			expect(this vector0 area, is equal to(-68.89f) within(0.01f))
+			expect(this vector0 length, is equal to(22.44f) within(0.01f))
+			expect(_empty empty, is equal to(true))
+			expect(_empty area, is equal to(0.0f) within(this precision))
+			//expect(this vector0 empty, is equal to(false)) //FIXME This crashes Fixture
```

Comment:
And FYI, it crashes because Cell toString has no overload for Bool, which we probably should fix.